src/main/native/glue/a/AaBox.cpp

```diff
@@ -43,6 +44,20 @@ JNIEXPORT jboolean JNICALL Java_com_github_stephengold_joltjni_AaBox_contains
     return result;
 }
 
+/*
+ * Class:     com_github_stephengold_joltjni_AaBox
+ * Method:    contains
+ * Signature: (JJ)Z
+ */
+JNIEXPORT jboolean JNICALL Java_com_github_stephengold_joltjni_AaBox_contains__JJ
+  (JNIEnv *, jclass, jlong boxVal, jlong inOther) {
```

Comment:
Rename boxVal -> boxVa. Argument names on the C++ side should ideally match those on the Java side.